docs/application/tizen-studio/setup/hardware-accelerated-execution-manager.md

```diff
@@ -97,15 +88,15 @@ If you meet an installation failure because of hardware requirements, check the
 <a name="on_MacOS"></a>
 ## Installing Intel&reg; HAXM on macOS
 
-Supported macOS versions: 10.10 Yosemite (64-bit), 10.9 Mavericks (64-bit), 10.8 Mountain Lion (64-bit)
+Supported macOS versions: 10.12 Sierra (64-bit), 10.11 OS X El Capitan(64-bit), 10.10 Yosemite (64-bit), 10.9 Mavericks (64-bit), 10.8 Mountain Lion (64-bit)
 
-**HAXM is installed automatically as part of the Tizen Studio.** If you want to install the HAXM individually, visit [http://download.tizen.org/sdk/haxm/6.0.3/mac/](http://download.tizen.org/sdk/haxm/6.0.3/mac/) and download the macOS installer package.
+**HAXM is installed automatically as part of the Tizen Studio.** If you want to install the HAXM individually, visit [http://download.tizen.org/sdk/haxm/7.0.0/mac/](http://download.tizen.org/sdk/haxm/7.0.0/mac/) and download the macOS installer package.
```

Comment:
**Rephrase** HAXM is installed automatically **to** HAXM is automatically installed

**Rephrase** If you want to install the HAXM individually **to** If you want to install the HAXM separately